packages/app-content-pages/server/server.js

```diff
@@ -4,15 +4,21 @@ if (process.env.NEWRELIC_LICENSE_KEY) {
 
 const express = require('express')
 const next = require('next')
+const https = require('https')
+const fs = require('fs')
 
 const setLogging = require('./set-logging')
 const setCacheHeaders = require('./set-cache-headers')
 
 const port = parseInt(process.env.PORT, 10) || 3000
+const hostname = 'local.zooniverse.org'
 const dev = process.env.NODE_ENV !== 'production'
 const app = next({ dev })
 const handle = app.getRequestHandler()
 
+const key = fs.readFileSync('server.key');
+const cert = fs.readFileSync('server.cert');
```

Comment:
This crashes if either `server.cert` or `server.key` don't exist, which they won't in a staging or production deploy. I need to read up some more on how Node filesystem commands work. 🤔

Reply:
Fixed with `fs.existsSync(path)` for both files.